package.json

```diff
@@ -33,6 +33,7 @@
     "ember-modifier": "^2.1.1"
   },
   "devDependencies": {
+    "@ember/legacy-built-in-components": "^0.2.0",
```

Comment:
@charlesfries this should be in `dependencies` as it's needed at run time when this addon is installed into host app

Reply:
Fixed